src/components/DateRangeFIlter/DateRangeFilter.tsx

```diff
@@ -34,7 +48,7 @@ export const DateRangeFilter = () => {
           ...old,
           [dateRangeOption]: {
             from: date?.from,
-            to: date?.to,
+            to: addDays(date!.to!, 1),
```

Comment:
At `from` line you're using `date?.` and one line below at `to` you're using `date!.`.
- Please take a look at this.

Reply:
Fixed